src/wp-includes/class-wp-theme-json.php

```diff
@@ -456,6 +456,7 @@ class WP_Theme_JSON {
 	public static function get_element_class_name( $element ) {
 		$class_name = '';
 
+		// Note: array_key_exists() should be replaced with an isset() check once WordPress drops support for PHP 5.6.
```

Comment:
Could a Trac ticket be opened for tracking this with a detailed explanation of why that can be linked to for additional context? At a minimum, a summary of the discussion on this PR into a comment on the current Trac ticket would also work.

Reply:
Opened https://core.trac.wordpress.org/ticket/57067 and updated all of these comments throughout the PR to point to it.